modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path

```diff
@@ -550,6 +550,11 @@
 	<td>//div[contains(@class,'side-panel-iframe-header')]//button[contains(@class,'side-panel-iframe-close')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CHECK_WAREHOUSE_COLLAPSE_OPEN</td>
+	<td>//a[contains(.,'${specifiedCollapse}')]/../../..//div[contains(@class,'show')]</td>
```

Comment:
Please use the following standard when putting vars into locators: ${key_varName}

e.g. ${key_specifiedCollapse}

Reply:
changes made